homeassistant/components/met/weather.py

```diff
@@ -132,17 +127,16 @@ def _init_data(self):
             # Retry in 15 to 20 minutes.
             minutes = 15 + randrange(6)
             _LOGGER.error("Retrying in %i minutes", minutes)
-            self._unsub_fetch_data = async_call_later(
-                self.hass, minutes * 60, self._fetch_data
-            )
+            self._unsub_fetch_data = async_call_later(self.hass, minutes * 60,
+                                                      self._fetch_data)
             return
 
         # Wait between 55-65 minutes. If people update HA on the hour, this
         # will make sure it will spread it out.
 
-        self._unsub_fetch_data = async_call_later(
-            self.hass, randrange(55, 65) * 60, self._fetch_data
-        )
+        self._unsub_fetch_data = async_call_later(self.hass,
+                                                  randrange(55, 65) * 60,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Standard pseudo-random generators are not suitable for security/cryptographic purposes.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4639533)